addons/website/views/res_config_settings_views.xml

```diff
@@ -59,7 +59,7 @@
                                     <label class="col-lg-3" string="Company" for="website_company_id"/>
                                     <field name="website_company_id" required="website_id" title="The company this website belongs to"/>
                                 </div>
-                                <div class="row mt8">
+                                <div class="row mt8" id="favicon_details">
```

Comment:
Hello @pash-odoo 

Following your recent changes, I guess this change is not needed anymore ?
```suggestion
                                <div class="row mt8">
```